mrp_bom_cust_analytics_valuation/models/mrp_unbuild_process_type.py

```diff
@@ -0,0 +1,34 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3.0 (https://www.gnu.org/licenses/agpl-3.0.html)
+
+from odoo import fields, models
+
+class MrpUnbuildProcessType(models.Model):
+    _inherit = "mrp.unbuild.process.type"
+
+    labour = fields.Monetary(
```

Comment:
Better naming and description IMO:
```suggestion
    cost_hr_manpower = fields.Monetary(
        string="Manpower unit cost",
```

Please apply the same criteria to the other fields